src/Spout/Writer/XLSX/Internal/Worksheet.php

```diff
@@ -212,8 +212,10 @@ private function getCellXML($rowIndex, $cellNumber, $cellValue, $styleId)
         $columnIndex = CellHelper::getCellIndexFromColumnIndex($cellNumber);
         $cellXML = '<c r="' . $columnIndex . $rowIndex . '"';
         $cellXML .= ' s="' . $styleId . '"';
-
-        if (CellHelper::isNonEmptyString($cellValue)) {
+        
+        if (CellHelper::isFormulaString($cellValue)) {
+            $cellXML .= '><f>'.substr($cellValue,1).'</f><v>0</v></c>'; // seriously, that's it.
```

Comment:
Also, by setting the value to `0`, reading back a spreadsheet with a formula won't work as Spout would return "0" for the cell value. Not ideal...